### PR TITLE
fix: Update mwcsys_executil.cpp

### PR DESCRIPTION
**Pull request by SaketKaswa20**
Removed unused header mwcsys_executil.h

*Issue number of the reported bug or feature request: #36 *

**Describe your changes**
Tried to remove the unused <#include mwcsys_executil.h> from the codebase

**Testing performed**
Describe the testing you have performed to ensure that the bug has been addressed, or that the new feature works as planned.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

**Comment by 678098** [comment]
@SaketKaswa20 
Now you can see PR being built with error "/home/runner/work/blazingmq/blazingmq/src/groups/mwc/mwcsys/mwcsys_executil.cpp:45:5: error: ‘ExecUtil’ has not been declared"

This happened because the removed header "linked" together 'mwcsys_executil.h' containing declarations and 'mwcsys_executil.cpp' containing definitions. More strictly, include directive instructs preprocessor to copy contents of the entire 'mwcsys_executil.h' file into the corresponding .cpp file on a preprocessor stage of compilation.
You can read more about here:
https://en.cppreference.com/w/cpp/preprocessor/include

In summary, removing this concrete include is not correct, we want to leave 'mwcsys_executil' files unchanged. You still need to check these files as I mentioned in the issue before removing the header from other files.

There is a link in the issue description with repo search which will help to find other files with this include: https://github.com/search?q=repo%3Abloomberg%2Fblazingmq%20ExecUtil&type=code
